src/core/endpoints/fetch_messages.js

```diff
@@ -9,7 +9,7 @@ function __processMessage(modules, message: Object): Object | null {
   if (!config.cipherKey) return message;
 
   try {
-    return crypto.decrypt(message);
+    return crypto.decrypt(message) || config.customDecrypt(message, config.cipherKey);
```

Comment:
similar as above, if we drive this into the crypto class, we don't need to worry about updating endpoints